utils/erp.py

```diff
@@ -60,6 +60,15 @@ def push_fr_data(data, retired=False):
     else:
         InitialRequestType, InitialRequestValue =  "Empty", 0
 
+    # In Early Warning / Early Action we use "Potentially Affected", not the normal _affected ones:
+    NumberOfPeopleAffected = \
+        (0 if data.num_affected is None else data.num_affected) + \
+        (0 if data.gov_num_affected is None else data.gov_num_affected) + \
+        (0 if data.other_num_affected is None else data.other_num_affected) + \
+        (0 if data.num_potentially_affected is None else data.num_potentially_affected) + \
+        (0 if data.gov_num_potentially_affected is None else data.gov_num_potentially_affected) + \
+        (0 if data.other_num_potentially_affected is None else data.other_num_potentially_affected)
```

Comment:
We can also do this.
```suggestion
    number_of_people_affected = sum([
        affected for affected in [
            data.num_affected,
            data.gov_num_affected,
            data.other_num_affected,
            data.num_potentially_affected,
            data.gov_num_potentially_affected,
            data.other_num_potentially_affect,
        ]
        if affected is not None
    ])
```